app/main.py

```
@@ -36,89 +124,80 @@ def process_csv(file_content: bytes, filename: str) -> None:
-    
-    Args:
-        limit (int): The number of users to return per page.
-        page (int): The page number to retrieve.
-
-    Returns:
-        UsersResponse: A response containing the list of users, total pages, and next page.
-    """
-    total_users = db.query(models.User).count()
-    total_pages = (total_users + limit - 1) // limit  # Calculate total pages
+async def get_users(
+    db: AsyncSession = Depends(get_db),
```

Ruff

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)